CHANGELOG.md

```diff
@@ -32,6 +32,10 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
   unless repo update was specifically requested.
   [Artem Sheremet](https://github.com/dotdoom)
 
+* Set Default Module for Storyboards in resource bundle targets.
```

Comment:
please add two empty spaces at the end of this line for formatting.

Reply:
Sorry about that. Its done now.